WordPress/Resources/release_notes.txt

```diff
@@ -4,7 +4,7 @@ Speaking of media, we know you might have given the app limited access to your m
 
 We also made VoiceOver and Dynamic Type more accurate on Activity Log and Schedule Post calendars. We’re happy to report that this improves the user experience for folks who need better visual accessibility.
 
-Big news for Comments: you can now follow a conversation using a button in the Comments header and moderate individual comments through a drop-down menu.
+Big news for Comments: you can now follow a conversation using a button in the Comments header.
```

Comment:
I only removed the final part of the sentence because the first refer to #17749, as per https://github.com/wordpress-mobile/WordPress-iOS/commit/1da4090242fb746307b7e86fc7689f6584baec1a#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bcR3